### PR TITLE
Fix to Remove * imports in subgraph, utils #461

### PR DESCRIPTION
**Pull request by dfrancisco1998**
#### What does this implement/fix? Explain your changes.
#461 fix removing import *, unsure if anything must be done for test files.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ jobs:
   include:
     - python: 3.6
     - python: 3.7
+    - python: 3.8
```
Comment (bdpedigo) [inline .travis.yml:10]: was this change supposed to be part of the PR?

Comment (dfrancisco1998) [reply .travis.yml:10]: @bdpedigo no but i am unsure of how to get rid of it. I even made a new branch.

Comment (bdpedigo) [reply .travis.yml:10]: need to remove this commit https://github.com/microsoft/graspologic/pull/483/commits/757e483934aff17c45952de80b9824d168a26b2e from this branch

Comment (dfrancisco1998) [reply .travis.yml:10]: @bdpedigo I am still unsure what i should do. I did git reset --hard 757e483. But it still appears in the file .travis.yml file. Should I have done one more commit before that?

Comment (bdpedigo) [reply .travis.yml:10]: @dfrancisco1998 part of sprint 0 is typically banging your head against the github wall for at least a little bit... i still `git` frustrated by these things too. mind posting your question in the slack discussion channel for everyone to see and try to help?

Comment (bdpedigo) [reply .travis.yml:10]: https://stackoverflow.com/a/40926981

**Pull request closed without merging**